src/components/VideoChatMenu.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popover from './Popover';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import MenuItem from './MenuItem';
+import openURLInNewTab from '../libs/openURLInNewTab';
+import ZoomIcon from '../../assets/images/zoom-icon.svg';
+import GoogleMeetIcon from '../../assets/images/google-meet.svg';
+import CONST from '../CONST';
+
+const propTypes = {
+    // State that determines whether to display the create menu or not
+    isVisible: PropTypes.bool.isRequired,
+
+    // Callback that determines behavior when menu is closed
+    onClose: PropTypes.func.isRequired,
+
+    ...windowDimensionsPropTypes,
+};
+
+const VideoChatMenu = (props) => {
+    const menuItemData = [
+        {
+            icon: ZoomIcon,
+            text: 'Zoom',
+            onPress: () => openURLInNewTab(CONST.NEW_ZOOM_MEETING_URL),
+        },
+        {
+            icon: GoogleMeetIcon,
+            text: 'Google Meet',
+            onPress: () => openURLInNewTab(CONST.NEW_GOOGLE_MEET_MEETING_URL),
+        },
+    ].map(item => ({
+        ...item,
+        onPress: () => {
+            item.onPress();
+            props.onClose();
+        },
+    }));
+
+    return (
+        <Popover
+            onClose={props.onClose}
+            isVisible={props.isVisible}
+            anchorPosition={{
```

Comment:
I see you went with an absolute position here ... what was your reasoning there? Maybe I can help you get it positioned relative to the video chat icon (which I believe Tim and I agreed would be a better approach).

To do that:

1) Convert the `HeaderView` into a class component, instead of the `ReportScreen`. Move `toggleVideoChatMenu` and `isVideoChatMenuActive` to the `HeaderView` instead of the `ReportScreen`.
2) Create a state property called `videoChatIconPosition`, and initialize it to: `{x: 0, y: 0}`. Also create a class property called `videoChatIconWrapper`, and initialize it to `null`
3) Create a class function called `measureVideoChatIconPosition` that looks like this:

```javascript
measureVideoChatIconPosition() {
    if (this.videoChatIconWrapper) {
        this.videoChatIconWrapper.measureInWindow((x, y) => this.setState({
            videoChatIconPosition: {x, y},
        }));
    }
}
```

4) Wrap the `Pressable` with the video icon in a `View` with the following props:

```jsx
<View
    ref={el => this.videoChatIconWrapper = el}
    onLayout={this.measureVideoChatIconPosition}
>
    ...
</View>
```

5) You'll then have the position of the video chat icon, and you can pass the `VideoChatMenu` an `anchorPosition` prop based off `this.state.videoChatIconPosition`.

Reply:
Oh crap. Honestly dude, I forgot to make the change to coordinate position before opening up to review. My bad. 

It was nice of you to spell out steps, but I also would appreciate a bit more breathing room. If for nothing else then the practice of implementing it myself.